examples/src/components/DynamicMeasuresExample.jsx

```diff
@@ -0,0 +1,193 @@
+import { AfmComponents } from '@gooddata/react-components';
+import * as GD from 'gooddata';
+
+import '@gooddata/react-components/styles/css/main.css';
+
+import { Layout } from './utils/Layout';
+import { Loading } from './utils/Loading';
+import { Error } from './utils/Error';
+import { SidebarItem } from './utils/SidebarItem';
+import {
+    monthDateIdentifier,
+    projectId,
+    franchiseFeesTag
+} from '../utils/fixtures';
+
+const { LineChart, ColumnChart } = AfmComponents;
+
+export class DynamicMeasuresExample extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            measureList: null,
+            error: null
+        };
+
+        this.onMeasureChange = this.onMeasureChange.bind(this);
+    }
+
+    componentWillMount() {
+        GD.xhr.get(`/gdc/md/${projectId}/tags/${franchiseFeesTag}`).then(
+            (response) => {
+                if (!response.entries.length) {
+                    return this.setState({
+                        measureList: null,
+                        error: {
+                            status: '404',
+                            message: `No measures with tag ${franchiseFeesTag}. Please check your project.
+                                Franchise fees measures should have assigned the tag ${franchiseFeesTag}.`
+                        }
+                    });
+                }
+                return this.setState({
+                    measureList: response.entries.map(entry => ({ ...entry, isSelected: true })),
+                    error: null
+                });
+            }
+        ).catch((error) => {
+            this.setState({
+                measureList: null,
+                error: { status: '400', message: `Error while requesting measures by tag ${franchiseFeesTag}. ${JSON.stringify(error)}` }
+            });
+        });
+    }
+
+    onMeasureChange(measureIdentifier) {
+        const { measureList } = this.state;
+        const updatedMeasure = measureList.find(measure => (measure.link === measureIdentifier));
+        const updatedMeasureIndex = measureList.indexOf(updatedMeasure);
+        const updatedMeasures = [...measureList];
+        updatedMeasures[updatedMeasureIndex] = {
+            ...updatedMeasure,
+            isSelected: !updatedMeasure.isSelected
+        };
+
+        this.setState({
+            measureList: updatedMeasures
+        });
+    }
+
+    getMeasureDefinition(measureItem) {
+        return {
+            localIdentifier: measureItem.link.split('/').reverse()[0],
+            definition: {
+                measure: {
+                    item: {
+                        uri: measureItem.link
+                    }
+                }
+            },
+            format: '#,##0'
+        };
+    }
+
+    render() {
+        const { measureList, error } = this.state;
+
+        if (error) {
+            return <Error error={{ status: '400', message: error }} />;
```

Comment:
`error` is an object with `{ status: string, message: string }` properties, I guess it will not render nicely